build/constants.yaml

```diff
@@ -244,6 +244,7 @@
     compressed-output-file-name: oryx_output.tar.gz
     build-manifest-file-name: oryx-manifest.toml
     os-type-file-name: .ostype
+    image-type-file-name: .imagetype
```

Comment:
@pauld-msft just double checking if we add the `ostype` and `imagetype` in the `oryx-manifest.toml` file? If not, this could be beneficial for our partners. The toml file stores build information meta data to be used between build and runtime images.

Reply:
As far as I know, we do not add those to the oryx-manifest.toml. However, we place the ostype and imagetype files in the exact same directory as the manifest file for our partners to consume and they are aware of these files.